nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/AbstractJsonEnricher.java

```diff
@@ -51,8 +55,12 @@ public boolean accept(Class<?> clazz, Type genericType, MediaType mediatype) {
     }
 
     @Override
-    public void write(Enriched<EntityType> enrichable, JsonGenerator jg) throws IOException {
-        write(jg, enrichable.getEntity());
+    public void write(Enriched<EntityType> enrichable, JsonGenerator jg) {
+        try {
+            write(jg, enrichable.getEntity());
+        } catch (IOException e) {
+            log.error(e);
```

Comment:
`log.error(e, e)` otherwise with what you wrote the stacktrace is not logged.
Also worth adding a `// TODO collect exception and return it to the caller`